tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,23 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    assert len(items) == 0
+    assert item_b not in items
+
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+# @pytest.mark.skip
+def test_get_no_catagoty_by_category():
```

Comment:
This should not be allowed to happen. We don't want to let users call the function with no category listed.